addons/blocks2image/userscript.js

```diff
@@ -0,0 +1,329 @@
+const buttonItem = [
```

Comment:
l10n

Reply:
That's very helpful, lol

Reply:
What he means is that you should set up localization. We can walk you through it, if you need.

Reply:
l10n has been added.  https://github.com/ScratchAddons/ScratchAddons/pull/2087/commits/f93b17a3f859c897b37d92ddee08ace93731fe88
Will Chinese be supported ?

Reply:
Chinese is not currently supported. If you would like to help translate to Chinese,  go to #1052